src/clarity-angular/data/stack-view/_stack-view.clarity.scss

```diff
@@ -197,36 +196,6 @@ $clr-stack-font-weight: clr-getTypePropertyValueForDomElement(stackview_text, fo
             height: 1rem;
         }
 
-        .stack-block-expandable > .stack-block-content {
```

Comment:
Why is this removed?

Reply:
Will mark this for cleanup instead of removing.